src/scripting/lua_game.cc

```diff
@@ -1093,7 +1099,7 @@ int LuaPlayer::get_produced_wares_count(lua_State* L) {
       necessarily mean that this player *can* attack the other player, as they might for
       example be in the same team.
 
-      :arg playernumber: An index in the :class:`array` of :attr:`~wl.bases.EditorGameBase.players`.
+      :arg playernumber: value of :attr:`wl.game.Player.number`.
```

Comment:
Since starting a sentence after a colon is usual in our documentation please do so as well. Same for all below.

Reply:
like "The value of XXX.number of the other player." ?

Reply:
Yes, something like that :-)